docs/authentication/hosted-add-authentication.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Add authentication
+id: hosted-add-authentication
+description: ""
+slug: /hosted-add-authentication
+keywords:
+  - scenario
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/07/2023
+  author: Anna Garcia
+draft: false
+hide_table_of_contents: false
+hide_title: false
+hide_breadcrumbs: false
+doc_type: quickstart
+displayed_sidebar: mainSidebar
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import MultiLanguageCodeBlock from '@site/src/components/MultiLanguageCodeBlock';
+
+import ProtocolOIDC from '../includes/_protocol_oidc.mdx';
+import PKCES256 from '../includes/_pkce_s256.mdx';
+import HostedWeb from '../includes/_hosted-web.mdx';
+import ClientTypeConfidential from '../includes/_client-type_confidential.mdx';
+import GrantTypeAuthorizationCode from '../includes/_grant-type_authorization-code.mdx';
+import TokenEndpointAuthMethodClientSecretBasic from '../includes/_token-endpoint-auth-method_client-secret-basic.mdx';
+import HWADiagram from '../includes/_hwa-diagram.mdx';
+
+This guide shows how to add authentication to your application using Hosted Web Authenticator integration model. Hosted Web handles passkey registration and authentication and offers an easy developer integration experience to your application.
```

Comment:
```suggestion
This guide shows how to add authentication to your application using the Hosted Web Authenticator integration model. Hosted Web handles passkey registration and authentication and offers an easy developer integration experience to your application.
```

Reply:
I also think there should be an add-on or additional sentence that explains _how_ it handles registration and authentication e.g. through a hosted web page.